x-pack/plugins/observability_ai_assistant/public/hooks/use_observability_ai_assistant_router.ts

```diff
@@ -47,43 +40,16 @@ export function useObservabilityAIAssistantRouter(): StatefulObservabilityAIAssi
       ...observabilityAIAssistantRouter,
       push: (...args) => {
         const next = link(...args);
-
-        history.push(next);
-      },
-      navigateToConversationsApp: (path, ...args) => {
-        const [_, route, routeParam] = path.split('/');
-
-        const sanitized = routeParam.replace('{', '').replace('}', '');
-
-        const pathKey = args[0]?.path;
-
-        if (typeof pathKey !== 'object') {
-          return;
-        }
-
-        if (Object.keys(pathKey).length === 0) {
-          navigateToApp('observabilityAIAssistant', {
-            path: route,
-          });
-          return;
-        }
-
-        if (Object.keys(pathKey).length === 1) {
-          navigateToApp('observabilityAIAssistant', {
-            // @ts-expect-error
-            path: `${route}/${pathKey[sanitized]}`,
-          });
-          return;
-        }
+        navigateToApp('observabilityAIAssistant', { path: next, replace: false });
       },
       replace: (path, ...args) => {
         const next = link(path, ...args);
-        history.replace(next);
+        navigateToApp('observabilityAIAssistant', { path: next, replace: true });
```

Comment:
nice, does this not cause a full page refresh?

Reply:
No. I was also a little worried about using this internally when the name suggests it should be used when navigating between apps. But I don't observe any downsides like full page refreshes.

Reply:
perfect

Reply:
Replace does mean you can't go use browser back anymore no?

Reply:
> Replace does mean you can't go use browser back anymore no?

Yes, that's the difference between push and replace